muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/reducers/base/status.ts

```diff
@@ -7,6 +7,29 @@
 import $ from '~/lib/jquery';
 import { ActionType } from "~/actions";
 
+export interface WhoAmIType {
```

Comment:
Would consider if "Type" suffix is necessary use at all for Interfaces. Typescript documentation clearly doesn't use those even with `type alias`. Of course if this has been habit long time, then I understand. Its not that important, but I just wanted to point it out.